lib/Parser/Parse.cpp

```diff
@@ -2252,7 +2301,26 @@ void Parser::AddModuleImportEntry(ModuleImportEntryList* importEntryList, IdentP
     importEntryList->Prepend(*importEntry);
 }
 
-void Parser::AddModuleExportEntry(ModuleExportEntryList* exportEntryList, IdentPtr importName, IdentPtr localName, IdentPtr exportName, IdentPtr moduleRequest)
+void Parser::CheckForDuplicateExportEntry(ModuleExportEntryList* exportEntryList, IdentPtr exportName)
+{
+    Js::PropertyId newPropertyId = EnsurePropertyId(exportName);
```

Comment:
It's a syntax error to export the same name twice. This method is checking to see if we already have an export entry in the list for the name exportName and if so, throwing an error.